### ISSUE TITLE
Release UI - only show 10 revisions in history by default

### ISSUE THREAD

**Issue by bartaz** [canonical-websites/snapcraft.io#1325]
As mentioned in https://github.com/canonical-websites/snapcraft.io/pull/1315#issuecomment-439329377 we'd like to show only 10 revisions in history panel by default, with a 'Show more releases...' link to reveal more.

**Comment by bartaz** [comment]
@therealjuan Would it be OK to make this UI only fix for now - meaning we limit to 10 but just among the limit we currently get from API.

This means that if snap has many revisions (more than fits into our 500 size first batch we have from API) users will not be able (yet) to 'show more...' from outside of this limit. To fix that (allow fetching more history from API) would require bigger changes to the logic and will make it a much bigger update.

**Comment by therealjuan** [comment]
Probably we need to look at a solution further down the line so yeah that makes sense

**Comment by bartaz** [comment]
Yes, I agree. Especially that having 'Show more...' link will make users expect to be able to load more history. So the question is - are we ok with a quick workaround (that may feel broken for some users of big snaps) or do we want to wait and do it together with actually loading history from API.

**Comment by therealjuan** [comment]
Probably will be a good idea to discuss with Store and see what they think about this?

**Comment by bartaz** [comment]
Sure, from store API point of view nothing stops us from loading more history. So it's purely UX design decision: is limiting to latest 10 (even without loading whole history) important enough to implement it quickly first.

**Comment by therealjuan** [comment]
Probably it will be better to wait and do this when we start loading history from the API so maybe we can do the UI fix for now with the limit we currently have (500).
